imap_processing/cdf/config/imap_swe_l1a_variable_attrs.yaml

```diff
@@ -0,0 +1,206 @@
+# <=== Coordinates ===>
+spin_angle:
+  CATDESC: 180 measurements per quarter cycle. See VAR_NOTES for details
+  FIELDNAM: Spin Angle
+  FILLVAL: -9223370000000000000
```

Comment:
I am wondering if there is a way to (and if it is a good idea to) define some of these constant numbers in a separate YAML file (e.g. `global_constants.yaml`) and reference them in our instrument-specific yaml files, instead of defining them multiple times within a file.

Reply:
I agree. I don't how easy it is to inherit other values from other yaml files. But would be nice to define some constant value in yaml file.

Reply:
I agree with both of you, but it's not really a thing in YAML unfortunately. My idea was to use a placeholder word (like "INTFILLVAL") that is then replaced by the code when the attributes are read in, but I kind of... don't like that so I never did it. There may be some yaml feature I don't know about that would solve this problem but at least at first look I couldn't turn anything up :(